gradle/publish-dists.gradle

```
@@ -0,0 +1,46 @@
+apply plugin: 'maven-publish'
```

A good test would be to inject dummy values for the creds and ensure it fails on refusal from bintray/artifactory but that the artifacts it was shipping are correct.